src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt

```diff
@@ -153,12 +205,13 @@ open class HiddenSecretsPlugin : Plugin<Project> {
         }
 
         /**
-         * Copy Cpp files from the lib to the Android project if they don't exist yet
+         * Copy Cpp files from the lib to the Android project
+         * @param clean whether to overwrite existing files
          */
-        fun copyCppFiles() {
+        fun copyCppFiles(clean: Boolean) {
```

Comment:
A parameter called `overwrite` would be clearer. And to not change too much the code and the actual behaviour a default value should be set. So the actual calls won't change
`        fun copyCppFiles(overwrite: Boolean = false) {}`